### PR TITLE
NETOBSERV-1466 NodeDirection / IfDirection

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

- ~Rename `FlowDirection` to `NodeDirection`,~ improved descriptions and restore `FlowDirection` label when using deduper merge
- Skip reinterpret field renaming since it's now done directly in eBPF
- Update console plugin fields / queries to plural

To setup deduper merge, simply add the following config in ebpf debug (or advanced) configuration:
```yaml
      advanced:
        env:
          DEDUPER_JUST_MARK: "false"
          DEDUPER_MERGE: "true"
```

## Dependencies

https://github.com/netobserv/netobserv-ebpf-agent/pull/275
https://github.com/netobserv/flowlogs-pipeline/pull/601
https://github.com/netobserv/network-observability-console-plugin/pull/482

## Followup

A followup to remove Duplicate field is available at: https://github.com/netobserv/network-observability-operator/pull/580

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

**Comment by jpinsonneau** [comment]
@jotak as discussed in last scrum, I have restored `FlowDirection` as label name to avoid migration.

File: controllers/consoleplugin/config/static-frontend-config.yaml [hunk 1]
```diff
@@ -381,23 +381,30 @@ columns:
     default: false
     width: 10
   - id: FlowDirection
-    name: Direction
-    tooltip: The direction of the flow observed at the Node observation point.
+    name: Node Direction
+    tooltip: The interpreted direction of the flow observed at the Node observation point.
     field: FlowDirection
-    filter: direction
+    filter: flowDirection
```
Comment (jotak) [inline controllers/consoleplugin/config/static-frontend-config.yaml:387]: shouldn't we call the filter "nodeDirection" ? And btw, other filters are in snake case so I guess it's flow_direction or node_direction

**Review by jotak — COMMENTED** [review]
just a small comment other than that lgtm!

File: controllers/consoleplugin/config/static-frontend-config.yaml (hunk 1 above)
Comment (jpinsonneau) [reply controllers/consoleplugin/config/static-frontend-config.yaml]: Sure, done in https://github.com/netobserv/network-observability-operator/pull/578/commits/553bfe230e596a0b3282d7b709032bcadc295d0f

**Pull request merged**